### ISSUE TITLE
jtmodules no longer a python package

### ISSUE THREAD

**Issue by scottberry** [TissueMAPS/TissueMAPS#73]
Since https://github.com/TissueMAPS/JtLibrary/commit/5e2913d944b4218a97000013fcbe2d8f809a735c
there is no longer the ability to access jterator modules in the form of a python package. This was a particularly useful feature for development, becaues it allowed stringing modules together in python scripts to test a new module in a pipeline independently of tissuemaps.

I would like to see jtmodules be reinstated as a python package (perhaps independently of jtlibrary).

**Comment by jluethi** [comment]
As most jtmodules require the jtlibrary anyway, I think it's better to have them unified so that onlz one package needs to be installed to develop modules. In order to call them from within a python environment, the jtlibrary/modules folder just needs to be added to the python path, e.g. by using
```
import sys
sys.path.append('home/ubuntu/jtlibrary/modules')
```

or by directly changing the $PYTHONPATH variable

**Comment by scottberry** [comment]
I agree that we only need one repository, however @riccardomurri mentioned that there is no problem having two python packages in a single repository. I think this is the best solution. the jtmodules package would not need to be imported by tissuemaps but could nonetheless be imported as an entry point for non-tissuemaps use (including, but not limited to, development)

Your suggestion is not ideal as it depends on the absolute path to the repository.
